BasicMusicPlayer/app/src/main/java/data/PlaylistManager.kt

```diff
@@ -0,0 +1,68 @@
+package data
+
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import data.artwork.PlaylistImager
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import java.util.concurrent.CountDownLatch
+
+// Manages the playlist creation
+class PlaylistManager {
+
+
+    suspend fun fetchFullPlaylist(): MutableList<PlaylistDetails> {
```

Comment:
Could you add comments explaining what the difference between the full and little methods are?